test/agent.js

```diff
@@ -649,7 +649,7 @@ test('filters', function (t) {
         setTimeout(function () {
           t.end()
           server.close()
-        }, 50)
+        }, 200)
```

Comment:
This test was failing in a race condition way without this change.  It's unclear if this change actually fixed the test, or if the internet weather on the GHA virtual machines changed.  Failures included below for posterity. 

```
not ok 241 plan != count
  ---
    operator: fail
    expected: 1
    actual:   0
    at: Timeout._onTimeout (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/test/agent.js:650:13)
    stack: |-
      Error: plan != count
          at Test.assert [as _assert] (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/node_modules/tape/lib/test.js:304:54)
          at Test.bound [as _assert] (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/node_modules/tape/lib/test.js:91:32)
          at Test.fail (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/node_modules/tape/lib/test.js:398:10)
          at Test.bound [as fail] (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/node_modules/tape/lib/test.js:91:32)
          at completeEnd (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/node_modules/tape/lib/test.js:243:18)
          at next (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/node_modules/tape/lib/test.js:217:13)
          at Test._end (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/node_modules/tape/lib/test.js:236:5)
          at Test.bound [as _end] (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/node_modules/tape/lib/test.js:91:32)
          at Test.end (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/node_modules/tape/lib/test.js:191:10)
          at Test.bound [as end] (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/node_modules/tape/lib/test.js:91:32)
          at Timeout._onTimeout (/home/runner/work/apm-agent-nodejs/apm-agent-nodejs/test/agent.js:650:13)
          at ontimeout (timers.js:427:11)
          at tryOnTimeout (timers.js:289:5)
          at listOnTimeout (timers.js:252:5)
          at Timer.processTimers (timers.js:212:10)
  ...


```

Reply:
Capturing this context in this issue: https://github.com/elastic/apm-agent-nodejs/issues/2027